packages/mjml-core/src/helpers/jsonToXML.js

```diff
@@ -1,16 +0,0 @@
-const jsonToXML = ({ tagName, attributes, children, content }) => {
```

Comment:
Can we keep the file name, export the helpler from the new file and throw a warning to import jsonToMJML instead of this file ?

Thanks :D